app/experimenter/experiments/migrations/0096_add_telemetry.py

```diff
@@ -10,21 +10,6 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AddField(
-            model_name="experiment",
-            name="telemetry_event_category",
-            field=models.CharField(blank=True, max_length=255, null=True),
-        ),
-        migrations.AddField(
-            model_name="experiment",
-            name="telemetry_event_method",
-            field=models.CharField(blank=True, max_length=255, null=True),
-        ),
-        migrations.AddField(
-            model_name="experiment",
-            name="telemetry_event_object",
-            field=models.CharField(blank=True, max_length=255, null=True),
-        ),
```

Comment:
Oh sorry I should've mentioned, the only place you don't remove it is in this file

Reply:
Fixed